frontend/workflows/serverexperimentation/src/start-experiment.tsx

```diff
@@ -69,20 +65,33 @@ const ExperimentDetails: React.FC<ExperimentDetailsProps> = ({
       inputProps: { defaultValue: undefined },
     },
     upstreamClusterTypeSelectionEnabled && {
-      name: "upstreamClusterType",
+      name: "faultInjectionCluster",
```

Comment:
This is why all of the experiment were injecting fault on upstream previously - we were using the wrong name here.